ta/pkcs11/src/processing_symm.c

```diff
@@ -149,34 +222,120 @@ allocate_tee_operation(struct pkcs11_session *session,
 	return tee2pkcs_error(res);
 }
 
+static enum pkcs11_rc hash_secret_helper(enum pkcs11_mechanism_id mech_id,
+					 struct pkcs11_object *obj,
+					 TEE_Attribute *tee_attr,
+					 void **ctx,
+					 size_t *object_size)
```

Comment:
I suggest to rename `object_size` to  `object_size_bits` for consistency.

Reply:
Done